### PR TITLE
make -db flag work

### LINKED ISSUE #131: fix database user input flag
## Summary

PR #129 by @wizsk added support for saving reports to the database. However, it is currently not activated due to `permuteArgs` behavior that caused some problem when db name and path was passed by the user.

This issue intends to fix that.

### PR DESCRIPTION
**Pull request by luca-patrignani**
# Important

**Please provide the requested parameters below to help us review your pull request.**

**Pull requests without a body or explanation will be rejected.**

## Describe your changes
I made a small change to `permuteArgs` to make the -db flag work. The fix is just awful but it works. 
## Issue ticket number and link

Closes #131

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added tests.
- [ ] I have run `make check` and there are no failures.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

I don't want to steal the job of @wizsk, but I just wanted to show a solution to the issue.

### CODE REVIEW

**Comment by wizsk** [comment]
hi @luca-patrignani 
no worries can add these line 
```go
			var optionName string
			if v[1] == '-' {
				optionName = v[2:]
			} else {
				optionName = v[1:]
			}
```

**Comment by luca-patrignani** [comment]
> hi @luca-patrignani no worries can add these line
> 
> ```go
> 			var optionName string
> 			if v[1] == '-' {
> 				optionName = v[2:]
> 			} else {
> 				optionName = v[1:]
> 			}
> ```

Is this for `--db` flag style, right?

**Comment by wizsk** [comment]
@luca-patrignani  yes

File: tcping.go [hunk 1]
```diff
@@ -238,7 +239,11 @@ func processUserInput(tcpStats *stats) {
 	if *outputJson {
 		tcpStats.printer = newJsonPrinter(*prettyJson)
 	} else {
-		tcpStats.printer = &planePrinter{}
+		if *outputDb != "" {
+			tcpStats.printer = newDb(args, *outputDb)
+		} else {
+			tcpStats.printer = &planePrinter{}
+		}
```
Comment (pouriyajamshidi) [inline tcping.go:246]: this is a bit more readable:

```go
	if *outputJson {
		tcpStats.printer = newJsonPrinter(*prettyJson)
	} else if *outputDb != "" {
		tcpStats.printer = newDb(args, *outputDb)
	} else {
		tcpStats.printer = &planePrinter{}
	}
```

**Review by pouriyajamshidi — CHANGES_REQUESTED** [review]
Hi @luca-patrignani,

Thanks once more for your contribution. Looks good to me and can get merged in after the final requested changes.

**Comment by pouriyajamshidi** [comment]
Thank you both @luca-patrignani  and @wizsk for your contributions!

If anyone of you guys are possibly looking for more, this issue https://github.com/pouriyajamshidi/tcping/issues/123 is also open to pick up. There is an open PR for it already but that is **far** from anything acceptable to go in and also there are no responses from the owner of it.

**Review by pouriyajamshidi — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**